umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_shared/instrument.clj

```diff
@@ -12,6 +13,10 @@
   (str "/gmi:MI_Metadata/gmi:acquisitionInformation/gmi:MI_AcquisitionInformation/gmi:instrument"
     "/eos:EOS_Instrument"))
 
+(def instrument-alternative-xpath
```

Comment:
Add a docstring.

Reply:
I was wondering about that, but the other, similar `def`'s in the file didn't have them ...

Reply:
Done.